mewarpx/tests/test_mwxrun.py

```diff
@@ -87,26 +78,6 @@ def test_capacitive_discharge_multigrid(caplog, name):
     while run.control.check_criteria():
         mwxrun.simulation.step()
 
-    #######################################################################
-    # Compare mwxrun data gathering to diagnostic output                  #
-    #######################################################################
-
-    # the edges are treated differently in our gather rho versus the yt
-    # dump so we only look at the interior here
-    rho_data = np.mean(mwxrun.get_gathered_rho_grid(
-        species_name='he_ions', include_ghosts=False
-    )[:,1:,0], axis=0)[2:-2]
-
-    ds = yt.load( "diags/fields/fields00010" )
-    grid_data = ds.covering_grid(
-        level=0, left_edge=ds.domain_left_edge, dims=ds.domain_dimensions
-    )
-    rho_data_yt = np.mean(
-        grid_data['rho_he_ions'].to_ndarray()[:,:,0], axis=0
-    )[2:-2]
-
-    assert np.allclose(rho_data, rho_data_yt, rtol=0.01)
-
```

Comment:
It would be useful to have a test for rz, 1d, 2d, and 3d that analyzes diag output to catch any unintended physics changes.

Reply:
We definitely should have tests like that, I agree. Not in this file though (which is just testing that execution works). I will add a physics test once the gating upstream PR is completed.